__tests__/integration.test.js

```diff
@@ -96,3 +96,30 @@ describe('/api/articles/:article_id', () => {
         })
     });
 });
+
+describe('/api/articles', () => {
+    test('GET 200: responds with an array of article objects', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length).toBe(articleData.length);
+            expect(body.length).toBe(13);
+            for (let i=0; i<body.length - 1; i++) {
+                expect(body[i].created_at > body[i + 1].created_at);
+            }
+            body.forEach(() => {
+                expect.objectContaining({
```

Comment:
I'm curious as to why you used .objectContaining() rather than .toMatchObject()! i havent seen it before!